gateway/src/otel/index.ts

```diff
@@ -319,7 +319,7 @@ function getBaseUrl({ baseUrl, writeToken }: OtelSettings): string | undefined {
   const regionMatch = /pylf_v\d_(us|eu)/.exec(writeToken)
   if (regionMatch) {
```

Comment:
it'll just skip this block when it doesn't match. i now see the log below: https://logfire-us.pydantic.dev/samuelcolvin/pydantic-ai-gateway?q=trace_id%3D%27019a112f8391fc29f59b31e5eee64404%27+AND+span_id%3D%271d81ff2d3a450a9a%27&spanId=1d81ff2d3a450a9a&traceId=019a112f8391fc29f59b31e5eee64404&since=2025-10-23T13%3A08%3A37.905000Z&until=2025-10-23T14%3A08%3A39.111000Z